app/view_models/nsm/v1/disbursement.rb

```diff
@@ -94,7 +94,7 @@ def pricing
       end
 
       def format_vat_rate
-        "#{(vat_rate * 100).to_i}%"
+        apply_vat == 'true' ? "#{(vat_rate * 100).to_i}%" : '0%'
```

Comment:
Maybe it's cleaner to have `vat_rate` return 0 unless `apply_vat` is set? Or do we only want to handle this condition in this 1 place?